### ISSUE TITLE
No clear documentation for enabling padding in FeatureExtractionPipeline

### ISSUE THREAD

**Issue by anruijian** [huggingface/transformers#21971]
### System Info

- `transformers` version: 4.26.0
- Platform: Linux-4.19.0-23-cloud-amd64-x86_64-with-debian-10.13
- Python version: 3.7.12
- Huggingface_hub version: 0.12.0
- PyTorch version (GPU?): 1.13.1+cu116 (True)
- Tensorflow version (GPU?): not installed (NA)
- Flax version (CPU?/GPU?/TPU?): not installed (NA)
- Jax version: not installed
- JaxLib version: not installed
- Using GPU in script?: <fill in>
- Using distributed or parallel set-up in script?: <fill in>

### Who can help?

@Narsil

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

The documentation for the [FeatureExtractionPipeline](https://huggingface.co/docs/transformers/v4.26.1/en/main_classes/pipelines#transformers.FeatureExtractionPipeline)  does not currently provide instructions on how to use the truncation and padding arguments. 

These arguments can be passed in the `tokenize_kwargs` parameter, which is parsed by the [self._sanitize_parameters](https://github.com/huggingface/transformers/blob/v4.26.1/src/transformers/pipelines/feature_extraction.py#L58) method. While the `truncation` argument can also be passed as a separate keyword argument, the `padding` argument can only be recognized if it is included in `tokenize_kwargs`.

To improve clarity, it would be beneficial for the documentation to explicitly state that the existence of `tokenize_kwargs` parameter for passing tokenizer arguments and add that only the `truncation` argument can be used as a keyword argument, while other tokenizer parameters should be included in `tokenize_kwargs`.

### Expected behavior

Below is the code to show that how padding should be used in FeatureExtractionPipeline.

```python
example = "After stealing money from the bank vault, the bank robber was seen fishing on the Mississippi river bank."
tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased", use_fast=False)

pipeline_without_padding_as_an_argument = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    return_tensors=True,
)
pipeline_with_padding_as_an_argument = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    padding="max_length",
    return_tensors=True,
)

pipeline_with_padding_in_kwarg = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenizer=tokenizer,
    tokenize_kwargs={"padding": "max_length"},
    return_tensors=True,
)

print(
    pipeline_without_padding_as_an_argument(example).shape
)  # torch.Size([1, 22, 768])
print(
    pipeline_with_padding_as_an_argument(example).shape
)  # torch.Size([1, 22, 768]) padding = max_length not working
print(
    pipeline_with_padding_in_kwarg(example).shape
)  # torch.Size([1, 512, 768]) padding = max_length working
```

**Comment by Narsil** [comment]
I simplified a bit for future readers:

```python
example = "After stealing money from the bank vault, the bank robber was seen fishing on the Mississippi river bank."

pipeline_without_padding_as_an_argument = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    return_tensors=True,
)
pipeline_with_padding_in_kwarg = pipeline(
    "feature-extraction",
    model="bert-base-uncased",
    tokenize_kwargs={"padding": "max_length"},
    return_tensors=True,
)
```

Also please note that `padding: "max_length"` makes unnecessary long tensors in most cases, slowing down the overal inference of your model.

I would refrain heavily from using it in a pipeline. `{"padding": True}` should be better.

**Comment by anruijian** [comment]
That makes sense. Do we need to make it clear in the documentation about how to use `tokenize_kwargs`?

**Comment by Narsil** [comment]
PRs to make it clearer are welcome for sure !

**Comment by anruijian** [comment]
@Narsil Submitted PR #22031 that adds the `tokenize_kwargs` definition in `FeatureExtrationPipeline`. Thanks!

**Comment by Narsil** [comment]
Thanks it looks great.
